.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,9 @@
+name: Pull Request
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  rust-checks:
+    uses: paastech-cloud/.github/.github/workflows/rust-ci.yml@main
```

Comment:
Why create a separate repo for the workflow, and the steps lack verbosity (include names). I think in our case it's better to put the workflow directly on the repo.

Reply:
I'm okay with the release